### PR TITLE
Consistent browserId

### PR DESCRIPTION
**Pull request by shirish87**
````
client.createWorker({
  os: 'Windows',
  os_version: '10',
  browser: 'Edge',
  browser_version: 'latest',
  url: 'http://www.example.com',
  timeout: 20
}, function(err, worker) {
  // fails with:
  // Uncaught Error: Validation Failed - `browser_version` invalid
});
````

### CODE REVIEW

**Comment by scottgonzalez** [comment]
Can you rename all of the existing `getBrowserId()` methods to `_getBrowserId()` and then add a `getBrowserId()` method to the base API client which returns the lowercase version?

**Comment by shirish87** [comment]
Done!

File: lib/api.js [hunk 1]
```diff
@@ -124,7 +124,7 @@ extend(ApiBaseClient.prototype, {
 
 	updateLatest: function(browsers) {
 		var latest = this.latest = {};
-		var getBrowserId = this.getBrowserId;
+		var getBrowserId = this.getBrowserId.bind(this);
```
Comment (scottgonzalez) [inline lib/api.js]: Why is this now being bound?

Comment (scottgonzalez) [inline lib/api.js]: Oh, I see why.

**Comment by scottgonzalez** [comment]
Thanks. Released in v1.3.2.

**Pull request closed without merging**